app/src/main/java/org/jellyfin/mobile/player/PlaybackMenus.kt

```diff
@@ -128,6 +149,12 @@ class PlaybackMenus(
         }
     }
 
+    private fun updateSubtitlesButton() {
+        subtitlesButton.isEnabled = subtitleCount > 0
```

Comment:
I think we want to hide the button when there's no subtitles instead of disabling it. Other apps do the same thing.

Reply:
That's a good point, yeah. Gonna do that.